src/components/CityColumnChart/CityColumnChart.scss

```diff
@@ -50,6 +60,12 @@
   }
 }
 
+.chartWrapper {
+  @include for-tablet-landscape-up {
+    margin-top: 24px;
+  }
+}
+
 :global(.cityColumnChartTooltipHeader) {
```

Comment:
Those particular indicators had the wrong formats, I've fixed those now. However, some other ones still have trouble (e.g. 'Residential building approvals and growth'). I'm going to leave fixing those because I believe one of the changes incoming in design review is to shift out format into a header for the Y axis - which, though it would likely fix this particular issue, is something that needs negotiating & making sure it makes sense across the board.